packages/react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -386,6 +386,8 @@ const Dropdown = React.forwardRef(
           )}
           <button
             type="button"
+            role="combobox" // eslint-disable-line jsx-a11y/role-has-required-aria-props
+            aria-controls={getMenuProps().id}
```

Comment:
could we circunvent adding the eslint-disable by addind the required aria-expanded attr?  or would that interfere with something? 
```suggestion
            role="combobox"
            aria-expanded={isOpen}
            aria-controls={getMenuProps().id}
```

Reply:
It has it under the hood. Its just that its coming from the downshift props through `{...toggleButtonProps}` so eslint thinks it doesn't have it. 
<img width="501" alt="Screen Shot 2023-03-30 at 5 40 43 PM" src="https://user-images.githubusercontent.com/12533409/228979365-3a075745-0094-497e-bdb1-124f6bc5dc03.png">

Reply:
nice! could we do `aria-expanded={toggleButtonProps['aria-expanded']}` then? or alternatively add a comment explaining that that attribute is already added by toggleButtonProps? I just want it to be understood that we're not choosing to ignore an a11y rule 😅

Reply:
True I'll add a better comment instead cause idk if adding double aria-expanded would mess it up

Reply:
@francinelucca added it